### ISSUE TITLE
Please add Bitrate of each video from the list aviable 

### ISSUE THREAD

**Issue by cfreds** [TeamNewPipe/NewPipe#4954]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
I would like to see the bitrate of the each video, on the list of resolution aviable 
Example: MPEG-4 1080p, 4.5 Mbps | MPEG-4 720p, 2.1Mbps

Optionally, also describe alternatives you've considered.
Also you can option to see bouth the MPEG-4 and VP9 video aviable on the list and not only MPEG-4 or VP9. 

#### Is your feature request related to a problem? 
My main problem (and Others users of the app) Internet speed, if could I see one video use 4.5Mbps and I watch at 1.5x ill know that cant play on because only have 5Mbps, so ill choose 720 instead 1080p and avoid buffering. 

#### How will you/everyone benefit from this feature?
Adding this features all users can know what speed is necessary to watch some video, and know if their bandwidth allow it

Here and Example (from ytmu**c vapp) 
![S01121-10503057](https://user-images.githubusercontent.com/49008136/99882795-a7a5cc80-2be8-11eb-8c7d-195d695de586.png)

**Comment by eladkarako** [comment]
for a whole mp4, average bit-rate is simply <code>file size / (number of minutes * 0.0075)</code> (that covers most 360p/720p) but for HLS (and DASH), the video is a playlist of segments (not consistent in size nor duration, but maybe just the first segment to play can be used to a estimated the bit-rate using the same formula),  
<details>
<summary>alternatives</summary>

There are tables for avarage bit-rates depending on the resolutions in:  
<a href="https://support.google.com/youtube/answer/2853702?hl=en">here</a>, <a href="https://www.circlehd.com/blog/how-to-calculate-video-file-size">here</a> and <a href="https://www.diva-portal.org/smash/get/diva2:832852/FULLTEXT01.pdf">here</a> <sup>(page 32)</sup>.  
I'm not sure if the video information request includes something useful for those cases, but the video API (which we don't use) has <a href="https://developers.google.com/youtube/v3/docs/videos#fileDetails.audioStreams[].bitrateBps">bitrateBps</a> though.
</details>

Any ideas?

**Comment by cfreds** [comment]
> for a whole mp4, average bit-rate is simply `file size / (number of minutes * 0.0075)` (that covers most 360p/720p) but for HLS (and DASH), the video is a playlist of segments (not consistent in size nor duration, but maybe just the first segment to play can be used to a estimated the bit-rate using the same formula),
> 
> alternatives
> Any ideas?
No. Each video has different bitrate. For example one video of nature 1080p will have different size of one only show words at 1080p like .ppx

**Comment by eladkarako** [comment]
> No. Each video has different bitrate. For example one video of nature 1080p will have different size of one only show words at 1080p like .ppx  

Yes it is pretty clear that the content of the scenes in the video also effects the efficiency of the H.264 compression algorithm, which eventually effects the file-size...
\*sign\*... so no lazy solutions... calculation it is!
